sig-index.md

```diff
@@ -6,6 +6,7 @@ Don't see a SIG here that fits what you're looking for? [Submit a Proposal](sig-
 * [AWS](sig-aws/README.md)
 * [Spinnaker-as-Code](sig-spinnaker-as-code/README.md)
```

Comment:
nit: Having sorted your addition into the correct place, would you please move this SIG down in the list.